src/SdfEntityCreator.cc

```diff
@@ -108,6 +108,30 @@ static math::Pose3d ResolveSdfPose(const sdf::SemanticPose &_semPose)
   return pose;
 }
 
+static std::optional<sdf::JointAxis> ResolveJointAxis(
+    const sdf::JointAxis *_unresolvedAxis)
```

Comment:
nit: Can we pass a const ref instead so we don't have to check if the input is a nullptr?

Reply:
Done